src/applications/vaos/appointment-list/components/RequestedAppointmentDetailsPage.jsx

```diff
@@ -187,7 +174,9 @@ export default function RequestedAppointmentDetailsPage() {
           {!!provider && (
             <span>
               {provider.name ||
-                (provider.providerName || provider.practiceName)}
+                (provider.providerName ||
+                  provider.practiceName ||
+                  provider.providerName)}
```

Comment:
`provider.practiceName` is repeated twice.
```suggestion
                (provider.providerName || provider.practiceName )}
```